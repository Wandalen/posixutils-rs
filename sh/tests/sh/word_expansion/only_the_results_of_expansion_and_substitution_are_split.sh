IFS=,
VAR='a,b,c'

echo x,y,${VAR},z,w
echo x,y,$(echo 'a,b,c'),z,w

