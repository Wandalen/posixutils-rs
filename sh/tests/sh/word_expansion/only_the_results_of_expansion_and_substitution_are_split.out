x,y,a b c,z,w
x,y,a b c,z,w
